### PR TITLE
Change Seurat help text

### PR DESCRIPTION
**Pull request by astrovsky01**
We no longer allow RDS output, so I am removing that to avoid confusion

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/seurat/seurat.xml [hunk 1]
```diff
@@ -164,7 +164,6 @@ types of single cell data. See the `Seurat Guided Clustering tutorial`_ for more
 
 Optionally you can choose to output
 
-    * Seurat RDS object (can use within R)
     * Rscript
```
Comment (nsoranzo) [inline tools/seurat/seurat.xml:167]: We don't produce the Rscript output any more as well, right?

Comment (astrovsky01) [reply tools/seurat/seurat.xml:167]: We do, it's an option to print alongside the graphs instead of separate

Comment (nsoranzo) [reply tools/seurat/seurat.xml:167]: You mean the "Show code alongside outputs?" option? Then I'd rephrase here, e.g. "R code to generate each output".

Comment (astrovsky01) [reply tools/seurat/seurat.xml]: Better?

**Review by nsoranzo — APPROVED** [review]
